Updating recommendation for publishing symbols (#10105)
dotnet/docs: docs/standard/library-guidance/nuget.md

</Project>
```

> > The downside to embedded symbol files is they increase the package size by about 30% for .NET libraries compiled using SDK-style projects. If package size is a concern, you should publish symbols in a symbol package instead.

Reviewer: ```suggestion
> The downside of embedding symbol files is that they increase the package size by about 30% for .NET libraries compiled using SDK-style projects. If package size is a concern, you should publish symbols in a symbol package instead.
```

> **✔️ CONSIDER** publishing symbols as a symbol package (snupkg) to NuGet.org

Reviewer 2: ```suggestion
**✔️ CONSIDER** publishing symbols as a symbol package (`*.snupkg`) to NuGet.org
```

> > Symbol packages (snupkg) provide developers a good on-demand debugging experience without bloating the main package size and impacting restore performance for those who don't intend to debug the NuGet package.

Reviewer 2: ```suggestion
> Symbol packages (`*.snupkg`) provide developers a good on-demand debugging experience without bloating the main package size and impacting restore performance for those who don't intend to debug the NuGet package.
```
